src/lang.js

```diff
@@ -120,15 +120,15 @@ const LANGS_CAN_CORRECT = [
 ]
 
 function getLangCode(lang) {
-  if (!lang) {
+  if (!lang.toString()) {
```

Comment:
The condition `!lang` is still required and we need to add another condition to judge if it is a string.
```js
!lang || typeof lang !== 'string'
```